src/components/radios/index.njk

```diff
@@ -204,6 +232,20 @@ Let users select a single option from a list.
         text: 'If true, radio will be disabled.'
       }
     ],
+    [
```

Comment:
I think we need `items.{}.value` as well.

Reply:
added